src/main/resources/format.yaml

```diff
@@ -3,6 +3,10 @@ repository: "xstefank/test-repo"
 statusUrl: "https://github.com/xstefank/test-repo/blob/master/.github/pull_request_template.md"
 
 format:
+  skipPatterns:
+    title: "((.*)?NO JIRA REQUIRED(.*)\\s?)|([^do not|don't]\\sskip.*template.*check(.*)\\s?)|([^do not|don't]\\sbypass.*template.*check(.*)\\s?)"
+    commit: "((.*)?NO JIRA REQUIRED(.*)\\s?)|([^do not|don't]\\sskip.*template.*check(.*)\\s?)|([^do not|don't]\\sbypass.*template.*check(.*)\\s?)"
+    description: "((.*)?NO JIRA REQUIRED(.*)\\s?)|([^do not|don't]\\sskip.*template.*check(.*)\\s?)|([^do not|don't]\\sbypass.*template.*check(.*)\\s?)"
```

Comment:
`([^do not|don't]\\sskip.*template.*check(.*)\\s?)` -- if this regex will be matched the Tyr won't be included right? So `"do not skip template check"` included in title/commit/description will actually skip the template check??